client/verta/verta/_registry/models.py

```diff
@@ -33,3 +45,17 @@ def with_workspace(self, workspace_name=None):
         new_list = copy.deepcopy(self)
         new_list._msg.workspace_name = workspace_name
         return new_list
+
+    def set_page_limit(self, msg, param):
+        msg.pagination.page_limit = param
+        return msg
+
+    def set_page_number(self, msg, param):
+        msg.pagination.page_number = param
+        return msg
+
+    def page_limit(self, msg):
+        return msg.pagination.page_limit
+
+    def page_number(self, msg):
+        return msg.pagination.page_number
```

Comment:
Interesting, I didn't realize these APIs used a nested pagination. I will need to follow up on this later to use a cleaner interface..